### PR TITLE
Remove dead code from System.Net.Security

### PR DESCRIPTION
**Pull request by satano**
PR addresses issue #17905, project **System.Net.Security**.

### CODE REVIEW

File: src/System.Net.Security/src/Resources/Strings.resx [hunk 1]
```diff
@@ -184,78 +240,9 @@
   <data name="net_invalid_enum" xml:space="preserve">
     <value>The specified value is not valid in the '{0}' enumeration.</value>
   </data>
-  <data name="event_EnumerateSecurityPackages" xml:space="preserve">
-    <value>Enumerating security packages:</value>
-  </data>
```
Comment (stephentoub) [inline src/System.Net.Security/src/Resources/Strings.resx]: Aren't all of these event_ resources used implicitly by the NetEventSource?

File: src/System.Net.Security/src/Resources/Strings.resx [hunk 2]
```diff
@@ -283,84 +270,6 @@
   <data name="security_ServiceNameCollection_EmptyServiceName" xml:space="preserve">
     <value>A service name must not be null or empty.</value>
   </data>
-  <data name="net_allocate_ssl_context_failed" xml:space="preserve">
-    <value>Failed to allocate SSL/TLS context, OpenSSL error - {0}.</value>
-  </data>
```
Comment (stephentoub) [inline src/System.Net.Security/src/Resources/Strings.resx]: Aren't lots of these used by the Unix build?

Comment (satano) [reply src/System.Net.Security/src/Resources/Strings.resx]: Yes, they are. But I do not see it in my build - I do not have unix platform. So I always wait for builds herem, check the failing logs and fix it.

File: src/System.Net.Security/src/Resources/Strings.resx (hunk 1 above)
Comment (satano) [reply src/System.Net.Security/src/Resources/Strings.resx]: What do you mean by "_implicitly_"? I looked for some names (`event_EnumerateSecurityPackages`, `event_CertsAfterFiltering`) in all sources and did not find them anywhere except `System.Net.Security`.

File: src/System.Net.Security/src/Resources/Strings.resx (hunk 2 above)
Comment (stephentoub) [reply src/System.Net.Security/src/Resources/Strings.resx]: > But I do not see it in my build - I do not have unix platform

You don't need it just to build; you can build everything on Windows, e.g. to build the Linux build of the assembly, do `msbuild /p:OSGroup=Linux` from the src directory.

File: src/System.Net.Security/src/Resources/Strings.resx (hunk 1 above)
Comment (stephentoub) [reply src/System.Net.Security/src/Resources/Strings.resx]: I mean `EventSource` constructs these resource names dynamically by appending "event_" to the name of the event.

File: src/System.Net.Security/src/Resources/Strings.resx (hunk 2 above)
Comment (satano) [reply src/System.Net.Security/src/Resources/Strings.resx]: Great. Thanks a lot.

File: src/System.Net.Security/src/Resources/Strings.resx [hunk 3]
```diff
@@ -184,78 +252,9 @@
...
-  <data name="event_RemoteCertDeclaredValid" xml:space="preserve">
-    <value>Remote certificate was verified as valid by the user.</value>
-  </data>
-  <data name="event_RemoteCertUserDeclaredInvalid" xml:space="preserve">
-    <value>Remote certificate was verified as invalid by the user.</value>
-  </data>
-  <data name="event_RemoteCertHasNoErrors" xml:space="preserve">
-    <value>Remote certificate has no errors.</value>
-  </data>
```
Comment (stephentoub) [inline src/System.Net.Security/src/Resources/Strings.resx]: These event_ entries all need to be put back.

File: src/System.Net.Security/src/Resources/Strings.resx [hunk 4]
```diff
@@ -169,6 +225,18 @@
...
+  </data>
+  <data name="net_generic_operation_failed" xml:space="preserve">
+    <value>Operation failed with error - {0}.</value>
+  </data>
+  <data name="net_ssl_decrypt_failed" xml:space="preserve">
+    <value>Decrypt failed with OpenSSL error - {0}.</value>
+  </data>
+  <data name="net_ssl_encrypt_failed" xml:space="preserve">
+    <value>Encrypt failed with OpenSSL error - {0}.</value>
+  </data>
```
Comment (stephentoub) [inline src/System.Net.Security/src/Resources/Strings.resx]: Why did these move from their original location?

**Pull request merged**